LayoutKitSampleApp/Benchmarks/FeedItemLayoutKitView.swift

```diff
@@ -24,6 +30,12 @@ class FeedItemLayoutKitView: UIView, DataBinder {
 
         let content = ContentLayout(title: data.contentTitle, domain: data.contentDomain)
         layout = FeedItemLayout(actionText: data.actionText, posterProfile: posterProfile, posterComment: data.posterComment, contentLayout: content, actorComment: data.actorComment)
+        
+        setNeedsLayout()
```

Comment:
This is actually the same as for [`FeedItamManualView#L123`](https://github.com/linkedin/LayoutKit/blob/master/LayoutKitSampleApp/Benchmarks/FeedItemManualView.swift#L123).

Do you think a comment like this would be sufficient?
```swift
// Assure that `layoutSubviews` is called
setNeedsLayout()
```